Pipfile

```diff
@@ -41,6 +41,7 @@ pytest-randomly = "==3.2.1"
 safety = "==1.8.5"
 tox = "==3.14.5"
 vyper = "==0.1.0b12"
+packaging = "==20.3"
```

Comment:
This dependency must be defined in setup.py, not here.

Reply:
I notice it is in setup.py, so remove here please. We have the convention now that all framework dependencies are only listed in setup.py